garak/probes/topic.py

```diff
@@ -92,7 +92,7 @@ def _gen_prompts(self, term):
     def __init__(self, config_root=_config):
         super().__init__(config_root)
 
-        self.data_dir = _config.transient.cache_dir / "resources" / "wn"
+        self.data_dir = _config.transient.cache_dir / "data" / "wn"
```

Comment:
What's our plan for cache dir structure?

Reply:
There is an example [here](https://github.com/leondz/garak/pull/916/files#diff-d0e6a61922e5a549cc4c628aabf0b38c7c8c7c707a83a4f367ca511cd46e609bR56-R61) where `data` prefix is used already for a `shared` file to maintain consistency, maybe this should stay `_config.transient.cache_dir / "data" / "wn"` and other locations that use `cache_dir` to store in a `tool` namespace should place that namespace under `_config.transient.cache_dir / "data"`.

Reply:
so far in `cache` I see:

* plugin cache
* wn
* visual_jailbreak data

It seems advantageous to be able to share artefacts across plugins, so I disprefer the idea of having dirs named based on single plugins -- in general (visual jailbreak refers also to the dataset name so no foul there)

I also dislike using the root for new artefacts directly (but this seems a pretty bland opinion :) )

I can also see that we have content that's used by plugins - currently all datasets - and content that's used by garak "core"

So we might like two top level dirs (not dead set on the names, not at all)

* `core` - things like plugin cache
* `datasets` - uh, datasets (`wn`, `visual_jailbreak`)

Reply:
`nltk_data` i think might benefit from having its own handler; it's also used standalone. Maybe defer choice on this until we move the nltk, HF handlers out to their own place (e.g. `resources`)